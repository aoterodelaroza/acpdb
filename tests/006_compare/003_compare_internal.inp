## check: 003_compare_internal.out -a1e-10
## delete: 003_compare_internal.db
## labels: regression quick

system rm -f 003_compare_internal.db
connect 003_compare_internal.db

insert method test
end
insert method test2
end

insert litref bibtex ../zz_source/bsip2.bib
insert set s22
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method test
end
insert set s22b
 description A description.
 litrefs bsip1 boys1970
 din ../zz_source/s22.din
 directory ../zz_source/s22_xyz
 method test
end

insert calc
 property_type energy_difference
 file ../zz_source/s22.list
 method test2
end
insert calc
 property_type energy_difference
 file ../zz_source/s22b.list
 method test2
end

delete evaluation test s22.h2o_h2o
delete evaluation test2 s22.nh3_nh3

compare
 source test2
 property_type energy_difference
 method test
end
