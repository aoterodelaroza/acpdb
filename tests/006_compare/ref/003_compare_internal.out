%% verbose
%% system rm -f 003_compare_internal.db
* SYSTEM: rm -f 003_compare_internal.db

%% connect 003_compare_internal.db
* CONNECT 

Disconnecting previous database (if connected) 
Connecting database file 003_compare_internal.db
Creating skeleton database 

%% insert method test
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test

%% insert method test2
* INSERT: insert data into the database (METHOD)
# INSERT METHOD test2

%% insert litref bibtex ../zz_source/bsip2.bib
* INSERT: insert data into the database (LITREF)
# INSERT LITREF bsip1
# INSERT LITREF boys1970
# INSERT LITREF s30l
# INSERT LITREF s12lb
# INSERT LITREF s12la
# INSERT LITREF risthaus2013
# INSERT LITREF xdmlarge
# INSERT LITREF johnson2009
# INSERT LITREF burns2017
# INSERT LITREF parker2015
# INSERT LITREF smith2016
# INSERT LITREF thanthiriwatte2010
# INSERT LITREF marshall2011
# INSERT LITREF burns2011
# INSERT LITREF bauza2013
# INSERT LITREF johnson2003
# INSERT LITREF wilke2009
# INSERT LITREF zhao2005
# INSERT LITREF zhao2005b
# INSERT LITREF grimme2007
# INSERT LITREF parthiban2001
# INSERT LITREF karton2011
# INSERT LITREF kozuch2013
# INSERT LITREF binkley1980
# INSERT LITREF gordon1982
# INSERT LITREF pietro1982
# INSERT LITREF dobbs1987a
# INSERT LITREF dobbs1987b
# INSERT LITREF dithcfield1971
# INSERT LITREF hehre1972
# INSERT LITREF hariharan1973
# INSERT LITREF hariharan1974
# INSERT LITREF gordon1980
# INSERT LITREF francl1982
# INSERT LITREF weigend2005
# INSERT LITREF weigend2006
# INSERT LITREF sto3g
# INSERT LITREF mini
# INSERT LITREF jensen1
# INSERT LITREF jensen2
# INSERT LITREF jensen3
# INSERT LITREF jensen4
# INSERT LITREF b3
# INSERT LITREF lyp
# INSERT LITREF dunning1
# INSERT LITREF dunning2
# INSERT LITREF grimme2010
# INSERT LITREF xdmbasis
# INSERT LITREF witte2016
# INSERT LITREF davidson1986
# INSERT LITREF andzelm1984
# INSERT LITREF kruse2012
# INSERT LITREF hbs66
# INSERT LITREF halogen
# INSERT LITREF dcacp
# INSERT LITREF qcp
# INSERT LITREF hfacp
# INSERT LITREF lasso
# INSERT LITREF osborne2000
# INSERT LITREF wb97xd1
# INSERT LITREF wb97xd2
# INSERT LITREF pepconf
# INSERT LITREF xdmhybrid
# INSERT LITREF xdm
# INSERT LITREF lcwpbe1
# INSERT LITREF lcwpbe2
# INSERT LITREF kalescky2014
# INSERT LITREF nbody1
# INSERT LITREF wateracp
# INSERT LITREF hf-3c
# INSERT LITREF kulik2016
# INSERT LITREF sure2015
# INSERT LITREF goerigk2013
# INSERT LITREF galano2006
# INSERT LITREF jensen2010
# INSERT LITREF hehre1970
# INSERT LITREF curtiss1999
# INSERT LITREF dill1975
# INSERT LITREF clark1983
# INSERT LITREF frisch1984
# INSERT LITREF rassolov1998
# INSERT LITREF mackie2011
# INSERT LITREF burns2013
# INSERT LITREF rezac2011
# INSERT LITREF dilabio2008
# INSERT LITREF mackie2008
# INSERT LITREF mackie2010
# INSERT LITREF torres2012
# INSERT LITREF dilabio2014
# INSERT LITREF vansanten2015
# INSERT LITREF goerigk2011
# INSERT LITREF aorss3
# INSERT LITREF aorss1
# INSERT LITREF aspirin
# INSERT LITREF clathrates
# INSERT LITREF aupi2
# INSERT LITREF pyricarbamate
# INSERT LITREF tmcomplex
# INSERT LITREF wallach
# INSERT LITREF junctions
# INSERT LITREF hb-piperidine
# INSERT LITREF 2nitrile
# INSERT LITREF chpi-pdxanthate
# INSERT LITREF digold
# INSERT LITREF abo3
# INSERT LITREF bipodal-carbamate
# INSERT LITREF oscil-friction
# INSERT LITREF tibond
# INSERT LITREF clathrate-hpr
# INSERT LITREF epxars-fepxas
# INSERT LITREF graphitestress-theo
# INSERT LITREF agtao3surf
# INSERT LITREF xdmchiral
# INSERT LITREF graphenemesa
# INSERT LITREF anales
# INSERT LITREF electrides
# INSERT LITREF xdmgold
# INSERT LITREF kevan
# INSERT LITREF burrito
# INSERT LITREF step
# INSERT LITREF agtao3
# INSERT LITREF xdmc9
# INSERT LITREF kaolin
# INSERT LITREF xdmbench
# INSERT LITREF ncisolids
# INSERT LITREF graphitestress
# INSERT LITREF xdmsolids
# INSERT LITREF energy-correction
# INSERT LITREF cij-partition
# INSERT LITREF supramolecular
# INSERT LITREF qhamgo
# INSERT LITREF ctc-gibbs
# INSERT LITREF gibbs2b
# INSERT LITREF gibbs2a
# INSERT LITREF sbasga2
# INSERT LITREF sbasga2-sbpga2
# INSERT LITREF aggase2
# INSERT LITREF cdal2se4
# INSERT LITREF lapla
# INSERT LITREF qtree
# INSERT LITREF variational
# INSERT LITREF sbpga2
# INSERT LITREF critic
# INSERT LITREF runwien
# INSERT LITREF b88
# INSERT LITREF s225
# INSERT LITREF s22
# INSERT LITREF s66
# INSERT LITREF s668
# INSERT LITREF g3
# INSERT LITREF s22b
# INSERT LITREF s66b
# INSERT LITREF kb49b
# INSERT LITREF kb49a
# INSERT LITREF p26
# INSERT LITREF l7
# INSERT LITREF hsg0
# INSERT LITREF ionichb
# INSERT LITREF gmtkn55
# INSERT LITREF gmtkn30
# INSERT LITREF bsr36
# INSERT LITREF pconf
# INSERT LITREF sconf
# INSERT LITREF water
# INSERT LITREF x40
# INSERT LITREF s8
# INSERT LITREF bde03
# INSERT LITREF aconf
# INSERT LITREF bhperi1
# INSERT LITREF bhperi2
# INSERT LITREF bhperi3
# INSERT LITREF bhperi4
# INSERT LITREF darc
# INSERT LITREF idisp
# INSERT LITREF s668b
# INSERT LITREF adim6
# INSERT LITREF a24
# INSERT LITREF karlsruhe1
# INSERT LITREF karlsruhe2
# INSERT LITREF pople1
# INSERT LITREF pople2
# INSERT LITREF pople3
# INSERT LITREF pople4
# INSERT LITREF pople5
# INSERT LITREF pople6
# INSERT LITREF pople7
# INSERT LITREF pople8
# INSERT LITREF bonifacic1974
# INSERT LITREF brauer2014
# INSERT LITREF clementi1982
# INSERT LITREF stevens1984
# INSERT LITREF topiol1980
# INSERT LITREF topiol1981
# INSERT LITREF duijneveldt1994
# INSERT LITREF davidson1994
# INSERT LITREF xdmpoly
# INSERT LITREF critic2
# INSERT LITREF kahn1972
# INSERT LITREF kahn1976
# INSERT LITREF haynes2006
# INSERT LITREF faver2011
# INSERT LITREF faver2012
# INSERT LITREF jensen2013
# INSERT LITREF goerigk2014
# INSERT LITREF cao2011
# INSERT LITREF dolg2011
# INSERT LITREF kulik2016b
# INSERT LITREF mentel2013
# INSERT LITREF mao2016
# INSERT LITREF deng2011
# INSERT LITREF gutowski1993
# INSERT LITREF xdmsurface
# INSERT LITREF carter2014
# INSERT LITREF linear1
# INSERT LITREF linear2
# INSERT LITREF linear3
# INSERT LITREF spackman2001
# INSERT LITREF linear4
# INSERT LITREF whitten1973
# INSERT LITREF dunlap1979
# INSERT LITREF feyereisen1993
# INSERT LITREF vahtras1993
# INSERT LITREF weigend2002
# INSERT LITREF sodt2006
# INSERT LITREF gpu1
# INSERT LITREF gpu2
# INSERT LITREF gpu3
# INSERT LITREF liechtenstein1995
# INSERT LITREF dudarev1998
# INSERT LITREF kabsch1976
# INSERT LITREF kaminsky2007
# INSERT LITREF maier2015
# INSERT LITREF hornak2006
# INSERT LITREF bsip2

%% insert set s22
* INSERT: insert data into the database (SET)
# INSERT SET s22
# INSERT STRUCTURE s22.nh3_nh3
# INSERT STRUCTURE s22.nh3_nh3_1
# INSERT STRUCTURE s22.nh3_nh3_2
# INSERT PROPERTY s22.nh3_nh3
# INSERT EVALUATION (method=test;property=s22.nh3_nh3)
# INSERT STRUCTURE s22.h2o_h2o
# INSERT STRUCTURE s22.h2o_h2o_1
# INSERT STRUCTURE s22.h2o_h2o_2
# INSERT PROPERTY s22.h2o_h2o
# INSERT EVALUATION (method=test;property=s22.h2o_h2o)
# INSERT STRUCTURE s22.h2co2_h2co2
# INSERT STRUCTURE s22.h2co2_h2co2_1
# INSERT STRUCTURE s22.h2co2_h2co2_2
# INSERT PROPERTY s22.h2co2_h2co2
# INSERT EVALUATION (method=test;property=s22.h2co2_h2co2)
# INSERT STRUCTURE s22.formamide_formamide
# INSERT STRUCTURE s22.formamide_formamide_1
# INSERT STRUCTURE s22.formamide_formamide_2
# INSERT PROPERTY s22.formamide_formamide
# INSERT EVALUATION (method=test;property=s22.formamide_formamide)
# INSERT STRUCTURE s22.uracil_uracil_hb
# INSERT STRUCTURE s22.uracil_uracil_hb_1
# INSERT STRUCTURE s22.uracil_uracil_hb_2
# INSERT PROPERTY s22.uracil_uracil_hb
# INSERT EVALUATION (method=test;property=s22.uracil_uracil_hb)
# INSERT STRUCTURE s22.pyridoxine_aminopyridine
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_1
# INSERT STRUCTURE s22.pyridoxine_aminopyridine_2
# INSERT PROPERTY s22.pyridoxine_aminopyridine
# INSERT EVALUATION (method=test;property=s22.pyridoxine_aminopyridine)
# INSERT STRUCTURE s22.adenine_thymine_wcc1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_1
# INSERT STRUCTURE s22.adenine_thymine_wcc1_2
# INSERT PROPERTY s22.adenine_thymine_wcc1
# INSERT EVALUATION (method=test;property=s22.adenine_thymine_wcc1)
# INSERT STRUCTURE s22.ch4_ch4
# INSERT STRUCTURE s22.ch4_ch4_1
# INSERT STRUCTURE s22.ch4_ch4_2
# INSERT PROPERTY s22.ch4_ch4
# INSERT EVALUATION (method=test;property=s22.ch4_ch4)
# INSERT STRUCTURE s22.c2h4_c2h4
# INSERT STRUCTURE s22.c2h4_c2h4_1
# INSERT STRUCTURE s22.c2h4_c2h4_2
# INSERT PROPERTY s22.c2h4_c2h4
# INSERT EVALUATION (method=test;property=s22.c2h4_c2h4)
# INSERT STRUCTURE s22.c6h6_ch4
# INSERT STRUCTURE s22.c6h6_ch4_1
# INSERT STRUCTURE s22.c6h6_ch4_2
# INSERT PROPERTY s22.c6h6_ch4
# INSERT EVALUATION (method=test;property=s22.c6h6_ch4)
# INSERT STRUCTURE s22.c6h6_c6h6_pd
# INSERT STRUCTURE s22.c6h6_c6h6_pd_1
# INSERT STRUCTURE s22.c6h6_c6h6_pd_2
# INSERT PROPERTY s22.c6h6_c6h6_pd
# INSERT EVALUATION (method=test;property=s22.c6h6_c6h6_pd)
# INSERT STRUCTURE s22.pyrazine_pyrazine
# INSERT STRUCTURE s22.pyrazine_pyrazine_1
# INSERT STRUCTURE s22.pyrazine_pyrazine_2
# INSERT PROPERTY s22.pyrazine_pyrazine
# INSERT EVALUATION (method=test;property=s22.pyrazine_pyrazine)
# INSERT STRUCTURE s22.uracil_uracil_stack
# INSERT STRUCTURE s22.uracil_uracil_stack_1
# INSERT STRUCTURE s22.uracil_uracil_stack_2
# INSERT PROPERTY s22.uracil_uracil_stack
# INSERT EVALUATION (method=test;property=s22.uracil_uracil_stack)
# INSERT STRUCTURE s22.indole_c6h6_stack
# INSERT STRUCTURE s22.indole_c6h6_stack_1
# INSERT STRUCTURE s22.indole_c6h6_stack_2
# INSERT PROPERTY s22.indole_c6h6_stack
# INSERT EVALUATION (method=test;property=s22.indole_c6h6_stack)
# INSERT STRUCTURE s22.adenine_thymine_stack
# INSERT STRUCTURE s22.adenine_thymine_stack_1
# INSERT STRUCTURE s22.adenine_thymine_stack_2
# INSERT PROPERTY s22.adenine_thymine_stack
# INSERT EVALUATION (method=test;property=s22.adenine_thymine_stack)
# INSERT STRUCTURE s22.c2h4_c2h2
# INSERT STRUCTURE s22.c2h4_c2h2_1
# INSERT STRUCTURE s22.c2h4_c2h2_2
# INSERT PROPERTY s22.c2h4_c2h2
# INSERT EVALUATION (method=test;property=s22.c2h4_c2h2)
# INSERT STRUCTURE s22.c6h6_h2o
# INSERT STRUCTURE s22.c6h6_h2o_1
# INSERT STRUCTURE s22.c6h6_h2o_2
# INSERT PROPERTY s22.c6h6_h2o
# INSERT EVALUATION (method=test;property=s22.c6h6_h2o)
# INSERT STRUCTURE s22.c6h6_nh3
# INSERT STRUCTURE s22.c6h6_nh3_1
# INSERT STRUCTURE s22.c6h6_nh3_2
# INSERT PROPERTY s22.c6h6_nh3
# INSERT EVALUATION (method=test;property=s22.c6h6_nh3)
# INSERT STRUCTURE s22.c6h6_hcn
# INSERT STRUCTURE s22.c6h6_hcn_1
# INSERT STRUCTURE s22.c6h6_hcn_2
# INSERT PROPERTY s22.c6h6_hcn
# INSERT EVALUATION (method=test;property=s22.c6h6_hcn)
# INSERT STRUCTURE s22.c6h6_c6h6_t
# INSERT STRUCTURE s22.c6h6_c6h6_t_1
# INSERT STRUCTURE s22.c6h6_c6h6_t_2
# INSERT PROPERTY s22.c6h6_c6h6_t
# INSERT EVALUATION (method=test;property=s22.c6h6_c6h6_t)
# INSERT STRUCTURE s22.indole_c6h6_t
# INSERT STRUCTURE s22.indole_c6h6_t_1
# INSERT STRUCTURE s22.indole_c6h6_t_2
# INSERT PROPERTY s22.indole_c6h6_t
# INSERT EVALUATION (method=test;property=s22.indole_c6h6_t)
# INSERT STRUCTURE s22.phenol_phenol
# INSERT STRUCTURE s22.phenol_phenol_1
# INSERT STRUCTURE s22.phenol_phenol_2
# INSERT PROPERTY s22.phenol_phenol
# INSERT EVALUATION (method=test;property=s22.phenol_phenol)

%% insert set s22b
* INSERT: insert data into the database (SET)
# INSERT SET s22b
# INSERT STRUCTURE s22b.nh3_nh3
# INSERT STRUCTURE s22b.nh3_nh3_1
# INSERT STRUCTURE s22b.nh3_nh3_2
# INSERT PROPERTY s22b.nh3_nh3
# INSERT EVALUATION (method=test;property=s22b.nh3_nh3)
# INSERT STRUCTURE s22b.h2o_h2o
# INSERT STRUCTURE s22b.h2o_h2o_1
# INSERT STRUCTURE s22b.h2o_h2o_2
# INSERT PROPERTY s22b.h2o_h2o
# INSERT EVALUATION (method=test;property=s22b.h2o_h2o)
# INSERT STRUCTURE s22b.h2co2_h2co2
# INSERT STRUCTURE s22b.h2co2_h2co2_1
# INSERT STRUCTURE s22b.h2co2_h2co2_2
# INSERT PROPERTY s22b.h2co2_h2co2
# INSERT EVALUATION (method=test;property=s22b.h2co2_h2co2)
# INSERT STRUCTURE s22b.formamide_formamide
# INSERT STRUCTURE s22b.formamide_formamide_1
# INSERT STRUCTURE s22b.formamide_formamide_2
# INSERT PROPERTY s22b.formamide_formamide
# INSERT EVALUATION (method=test;property=s22b.formamide_formamide)
# INSERT STRUCTURE s22b.uracil_uracil_hb
# INSERT STRUCTURE s22b.uracil_uracil_hb_1
# INSERT STRUCTURE s22b.uracil_uracil_hb_2
# INSERT PROPERTY s22b.uracil_uracil_hb
# INSERT EVALUATION (method=test;property=s22b.uracil_uracil_hb)
# INSERT STRUCTURE s22b.pyridoxine_aminopyridine
# INSERT STRUCTURE s22b.pyridoxine_aminopyridine_1
# INSERT STRUCTURE s22b.pyridoxine_aminopyridine_2
# INSERT PROPERTY s22b.pyridoxine_aminopyridine
# INSERT EVALUATION (method=test;property=s22b.pyridoxine_aminopyridine)
# INSERT STRUCTURE s22b.adenine_thymine_wcc1
# INSERT STRUCTURE s22b.adenine_thymine_wcc1_1
# INSERT STRUCTURE s22b.adenine_thymine_wcc1_2
# INSERT PROPERTY s22b.adenine_thymine_wcc1
# INSERT EVALUATION (method=test;property=s22b.adenine_thymine_wcc1)
# INSERT STRUCTURE s22b.ch4_ch4
# INSERT STRUCTURE s22b.ch4_ch4_1
# INSERT STRUCTURE s22b.ch4_ch4_2
# INSERT PROPERTY s22b.ch4_ch4
# INSERT EVALUATION (method=test;property=s22b.ch4_ch4)
# INSERT STRUCTURE s22b.c2h4_c2h4
# INSERT STRUCTURE s22b.c2h4_c2h4_1
# INSERT STRUCTURE s22b.c2h4_c2h4_2
# INSERT PROPERTY s22b.c2h4_c2h4
# INSERT EVALUATION (method=test;property=s22b.c2h4_c2h4)
# INSERT STRUCTURE s22b.c6h6_ch4
# INSERT STRUCTURE s22b.c6h6_ch4_1
# INSERT STRUCTURE s22b.c6h6_ch4_2
# INSERT PROPERTY s22b.c6h6_ch4
# INSERT EVALUATION (method=test;property=s22b.c6h6_ch4)
# INSERT STRUCTURE s22b.c6h6_c6h6_pd
# INSERT STRUCTURE s22b.c6h6_c6h6_pd_1
# INSERT STRUCTURE s22b.c6h6_c6h6_pd_2
# INSERT PROPERTY s22b.c6h6_c6h6_pd
# INSERT EVALUATION (method=test;property=s22b.c6h6_c6h6_pd)
# INSERT STRUCTURE s22b.pyrazine_pyrazine
# INSERT STRUCTURE s22b.pyrazine_pyrazine_1
# INSERT STRUCTURE s22b.pyrazine_pyrazine_2
# INSERT PROPERTY s22b.pyrazine_pyrazine
# INSERT EVALUATION (method=test;property=s22b.pyrazine_pyrazine)
# INSERT STRUCTURE s22b.uracil_uracil_stack
# INSERT STRUCTURE s22b.uracil_uracil_stack_1
# INSERT STRUCTURE s22b.uracil_uracil_stack_2
# INSERT PROPERTY s22b.uracil_uracil_stack
# INSERT EVALUATION (method=test;property=s22b.uracil_uracil_stack)
# INSERT STRUCTURE s22b.indole_c6h6_stack
# INSERT STRUCTURE s22b.indole_c6h6_stack_1
# INSERT STRUCTURE s22b.indole_c6h6_stack_2
# INSERT PROPERTY s22b.indole_c6h6_stack
# INSERT EVALUATION (method=test;property=s22b.indole_c6h6_stack)
# INSERT STRUCTURE s22b.adenine_thymine_stack
# INSERT STRUCTURE s22b.adenine_thymine_stack_1
# INSERT STRUCTURE s22b.adenine_thymine_stack_2
# INSERT PROPERTY s22b.adenine_thymine_stack
# INSERT EVALUATION (method=test;property=s22b.adenine_thymine_stack)
# INSERT STRUCTURE s22b.c2h4_c2h2
# INSERT STRUCTURE s22b.c2h4_c2h2_1
# INSERT STRUCTURE s22b.c2h4_c2h2_2
# INSERT PROPERTY s22b.c2h4_c2h2
# INSERT EVALUATION (method=test;property=s22b.c2h4_c2h2)
# INSERT STRUCTURE s22b.c6h6_h2o
# INSERT STRUCTURE s22b.c6h6_h2o_1
# INSERT STRUCTURE s22b.c6h6_h2o_2
# INSERT PROPERTY s22b.c6h6_h2o
# INSERT EVALUATION (method=test;property=s22b.c6h6_h2o)
# INSERT STRUCTURE s22b.c6h6_nh3
# INSERT STRUCTURE s22b.c6h6_nh3_1
# INSERT STRUCTURE s22b.c6h6_nh3_2
# INSERT PROPERTY s22b.c6h6_nh3
# INSERT EVALUATION (method=test;property=s22b.c6h6_nh3)
# INSERT STRUCTURE s22b.c6h6_hcn
# INSERT STRUCTURE s22b.c6h6_hcn_1
# INSERT STRUCTURE s22b.c6h6_hcn_2
# INSERT PROPERTY s22b.c6h6_hcn
# INSERT EVALUATION (method=test;property=s22b.c6h6_hcn)
# INSERT STRUCTURE s22b.c6h6_c6h6_t
# INSERT STRUCTURE s22b.c6h6_c6h6_t_1
# INSERT STRUCTURE s22b.c6h6_c6h6_t_2
# INSERT PROPERTY s22b.c6h6_c6h6_t
# INSERT EVALUATION (method=test;property=s22b.c6h6_c6h6_t)
# INSERT STRUCTURE s22b.indole_c6h6_t
# INSERT STRUCTURE s22b.indole_c6h6_t_1
# INSERT STRUCTURE s22b.indole_c6h6_t_2
# INSERT PROPERTY s22b.indole_c6h6_t
# INSERT EVALUATION (method=test;property=s22b.indole_c6h6_t)
# INSERT STRUCTURE s22b.phenol_phenol
# INSERT STRUCTURE s22b.phenol_phenol_1
# INSERT STRUCTURE s22b.phenol_phenol_2
# INSERT PROPERTY s22b.phenol_phenol
# INSERT EVALUATION (method=test;property=s22b.phenol_phenol)

%% insert calc
* INSERT: insert data into the database (CALC)
# INSERT EVALUATION (method=test2;property=22;nvalue=1)
# INSERT EVALUATION (method=test2;property=21;nvalue=1)
# INSERT EVALUATION (method=test2;property=20;nvalue=1)
# INSERT EVALUATION (method=test2;property=19;nvalue=1)
# INSERT EVALUATION (method=test2;property=18;nvalue=1)
# INSERT EVALUATION (method=test2;property=17;nvalue=1)
# INSERT EVALUATION (method=test2;property=16;nvalue=1)
# INSERT EVALUATION (method=test2;property=15;nvalue=1)
# INSERT EVALUATION (method=test2;property=14;nvalue=1)
# INSERT EVALUATION (method=test2;property=1;nvalue=1)
# INSERT EVALUATION (method=test2;property=2;nvalue=1)
# INSERT EVALUATION (method=test2;property=3;nvalue=1)
# INSERT EVALUATION (method=test2;property=4;nvalue=1)
# INSERT EVALUATION (method=test2;property=5;nvalue=1)
# INSERT EVALUATION (method=test2;property=6;nvalue=1)
# INSERT EVALUATION (method=test2;property=7;nvalue=1)
# INSERT EVALUATION (method=test2;property=8;nvalue=1)
# INSERT EVALUATION (method=test2;property=9;nvalue=1)
# INSERT EVALUATION (method=test2;property=10;nvalue=1)
# INSERT EVALUATION (method=test2;property=11;nvalue=1)
# INSERT EVALUATION (method=test2;property=12;nvalue=1)
# INSERT EVALUATION (method=test2;property=13;nvalue=1)

%% insert calc
* INSERT: insert data into the database (CALC)
# INSERT EVALUATION (method=test2;property=44;nvalue=1)
# INSERT EVALUATION (method=test2;property=43;nvalue=1)
# INSERT EVALUATION (method=test2;property=42;nvalue=1)
# INSERT EVALUATION (method=test2;property=41;nvalue=1)
# INSERT EVALUATION (method=test2;property=40;nvalue=1)
# INSERT EVALUATION (method=test2;property=39;nvalue=1)
# INSERT EVALUATION (method=test2;property=38;nvalue=1)
# INSERT EVALUATION (method=test2;property=37;nvalue=1)
# INSERT EVALUATION (method=test2;property=36;nvalue=1)
# INSERT EVALUATION (method=test2;property=23;nvalue=1)
# INSERT EVALUATION (method=test2;property=24;nvalue=1)
# INSERT EVALUATION (method=test2;property=25;nvalue=1)
# INSERT EVALUATION (method=test2;property=26;nvalue=1)
# INSERT EVALUATION (method=test2;property=27;nvalue=1)
# INSERT EVALUATION (method=test2;property=28;nvalue=1)
# INSERT EVALUATION (method=test2;property=29;nvalue=1)
# INSERT EVALUATION (method=test2;property=30;nvalue=1)
# INSERT EVALUATION (method=test2;property=31;nvalue=1)
# INSERT EVALUATION (method=test2;property=32;nvalue=1)
# INSERT EVALUATION (method=test2;property=33;nvalue=1)
# INSERT EVALUATION (method=test2;property=34;nvalue=1)
# INSERT EVALUATION (method=test2;property=35;nvalue=1)

%% delete evaluation test s22.h2o_h2o
* DELETE: delete data from the database (EVALUATION)

# DELETE EVALUATION (method=test;property=s22.h2o_h2o)

%% delete evaluation test2 s22.nh3_nh3
* DELETE: delete data from the database (EVALUATION)

# DELETE EVALUATION (method=test2;property=s22.nh3_nh3)

%% compare
* COMPARE: compare data to database evaluations

# -- Evaluation of data from method -- 
# Approximate method: test2
# Property type: ENERGY_DIFFERENCE
# Reference method: test
# Statistics: (partial, missing: 1 from source, 1 from reference)
# s22              rms = 26480.44884802   mae = 24151.48809500   mse = -24151.48809500  wrms = 118424.16739776  ndat = 20
# s22b             rms = 25271.09274754   mae = 22269.28382136   mse = -22269.28382136  wrms = 118531.93169103  ndat = 22
# ALL              rms = 25854.03268455   mae = 23165.57157071   mse = -23165.57157071  wrms = 167553.28183673  ndat = 42
#
Id Name                                       Approx_method         Ref_method         difference
1  s22.h2co2_h2co2                           -6275.0947000000     -18.7530000000   -6256.3417000000
2  s22.formamide_formamide                   -8157.6231100000     -16.0620000000   -8141.5611100000
3  s22.uracil_uracil_hb                     -10040.1515200000     -20.6410000000  -10019.5105200000
4  s22.pyridoxine_aminopyridine             -11922.6799300000     -16.9340000000  -11905.7459300000
5  s22.adenine_thymine_wcc1                 -13805.2083400000     -16.6600000000  -13788.5483400000
6  s22.ch4_ch4                              -15687.7367500000      -0.5270000000  -15687.2097500000
7  s22.c2h4_c2h4                            -17570.2651600000      -1.4720000000  -17568.7931600000
8  s22.c6h6_ch4                             -19452.7935700000      -1.4480000000  -19451.3455700000
9  s22.c6h6_c6h6_pd                         -21335.3219800000      -2.6540000000  -21332.6679800000
10 s22.pyrazine_pyrazine                    -23217.8503900000      -4.2550000000  -23213.5953900000
11 s22.uracil_uracil_stack                  -25100.3788000000      -9.8050000000  -25090.5738000000
12 s22.indole_c6h6_stack                    -26982.9072100000      -4.5240000000  -26978.3832100000
13 s22.adenine_thymine_stack                -28865.4356200000     -11.7300000000  -28853.7056200000
14 s22.c2h4_c2h2                            -30747.9640300000      -1.4960000000  -30746.4680300000
15 s22.c6h6_h2o                             -32630.4924400000      -3.2750000000  -32627.2174400000
16 s22.c6h6_nh3                             -34513.0208500000      -2.3120000000  -34510.7088500000
17 s22.c6h6_hcn                             -36395.5492600000      -4.5410000000  -36391.0082600000
18 s22.c6h6_c6h6_t                          -38278.0776700000      -2.7170000000  -38275.3606700000
19 s22.indole_c6h6_t                        -40160.6060800000      -5.6270000000  -40154.9790800000
20 s22.phenol_phenol                        -42043.1344900000      -7.0970000000  -42036.0374900000
21 s22b.nh3_nh3                              -2510.0378800000      -3.1330000000   -2506.9048800000
22 s22b.h2o_h2o                              -4392.5662900000      -4.9890000000   -4387.5772900000
23 s22b.h2co2_h2co2                          -6275.0947000000     -18.7530000000   -6256.3417000000
24 s22b.formamide_formamide                  -8157.6231100000     -16.0620000000   -8141.5611100000
25 s22b.uracil_uracil_hb                    -10040.1515200000     -20.6410000000  -10019.5105200000
26 s22b.pyridoxine_aminopyridine            -11922.6799300000     -16.9340000000  -11905.7459300000
27 s22b.adenine_thymine_wcc1                -13805.2083400000     -16.6600000000  -13788.5483400000
28 s22b.ch4_ch4                             -15687.7367500000      -0.5270000000  -15687.2097500000
29 s22b.c2h4_c2h4                           -17570.2651600000      -1.4720000000  -17568.7931600000
30 s22b.c6h6_ch4                            -19452.7935700000      -1.4480000000  -19451.3455700000
31 s22b.c6h6_c6h6_pd                        -21335.3219800000      -2.6540000000  -21332.6679800000
32 s22b.pyrazine_pyrazine                   -23217.8503900000      -4.2550000000  -23213.5953900000
33 s22b.uracil_uracil_stack                 -25100.3788000000      -9.8050000000  -25090.5738000000
34 s22b.indole_c6h6_stack                   -26982.9072100000      -4.5240000000  -26978.3832100000
35 s22b.adenine_thymine_stack               -28865.4356200000     -11.7300000000  -28853.7056200000
36 s22b.c2h4_c2h2                           -30747.9640300000      -1.4960000000  -30746.4680300000
37 s22b.c6h6_h2o                            -32630.4924400000      -3.2750000000  -32627.2174400000
38 s22b.c6h6_nh3                            -34513.0208500000      -2.3120000000  -34510.7088500000
39 s22b.c6h6_hcn                            -36395.5492600000      -4.5410000000  -36391.0082600000
40 s22b.c6h6_c6h6_t                         -38278.0776700000      -2.7170000000  -38275.3606700000
41 s22b.indole_c6h6_t                       -40160.6060800000      -5.6270000000  -40154.9790800000
42 s22b.phenol_phenol                       -42043.1344900000      -7.0970000000  -42036.0374900000
## The following properties are missing from the REFERENCE:
## s22.h2o_h2o
## The following properties are missing from the APPROX_METHOD:
## s22.nh3_nh3

